app/mailers/read_poem_mailer.rb

```diff
@@ -0,0 +1,7 @@
+class ReadPoemMailer < ApplicationMailer
+  def read_poem(user, poem)
```

Comment:
__[rails_best_practices]__
* use multipart/alternative as content_type of email [:ref](http://rails-bestpractices.com/posts/2010/08/05/use-multipart-alternative-as-content_type-of-email/)